core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/BasicPathShadow2afp.java

```diff
@@ -209,83 +207,36 @@ private void discretizePathIterator(
             case LINE_TO:
                 endx = element.getToX();
                 endy = element.getToY();
-                crossSegmentTwoShadowLines(
-                        curx, cury,
-                        endx, endy,
-                        x1, y1, x2, y2);
-                if (this.crossings == MathConstants.SHAPE_INTERSECTS) {
-                    return;
-                }
+                setLineToFromDiscretizePathIterator(curx, cury, endx, endy, x1, y1, x2, y2);
```

Comment:
The replaced code was supposed to exit this function if crossings is not zero. The replacement function does not provide the same feature.